src/JWT.ts

```diff
@@ -144,6 +144,7 @@ export interface PublicKeyTypes {
 }
 
 export const SUPPORTED_PUBLIC_KEY_TYPES: PublicKeyTypes = {
+  ES256: ['JsonWebKey2020', 'EcdsaSecp256r1VerificationKey2022'],
```

Comment:
Thanks for catching this. Having one untested was bothersome in the back of my mind. Removing one of them removed some of the pressure. I thought I had ``EcdsaSecp256r1VerificationKey2022`` in there for some reason, perhaps with VerificationAlgorithm.ts? Anyway, it may likely be a non-issue since all the tests pass (unless I didn't include some necessary ones).

Reply:
No problem, thanks for contributing this!

Algorithms or verification method types, like EcdsaSecp256r1VerificationKey2022, need to be defined somewhere as a standard and not just locally as code, so that other people can implement them.

Luckily we have the JsonWebKey2020 that covers a broad range of key types, so we don't need to define new ones.